pkg/cmd/attestation/trustedroot/trustedroot.go

```diff
@@ -116,6 +116,12 @@ func getTrustedRoot(makeTUF tufClientInstantiator, opts *Options) error {
 	// Disable local caching, so we get up-to-date response from TUF repository
 	tufOpt.CacheValidity = 0
 
+	// Target will be either the default trusted root, or the trust domain-qualified one
+	ghTR := defaultTR
+	if opts.TrustDomain != "" {
+		ghTR = fmt.Sprintf("%s.%s", opts.TrustDomain, defaultTR)
+	}
+
```

Comment:
One subtle change from the existing logic: instead of retrieving both the default target (`trusted_root.json`) AND the tenant-prefixed target (`some-trust-domain.trusted_root.json`), the new logic will retrieve ONLY the tenant-prefixed version if the `--hostname` flag is supplied.

If you're identifying a specific tenant with the `--hostname` flag it seems reasonable to assume that you're only interested in the trusted root associated with that tenant.

Reply:
Seems fair. PGI is always included though. Given that a properly configured policy is configured, there would be no risk of confusion here during verification.